apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -138,6 +142,35 @@ private string ValidatedDataSource(string dataSource)
             return dataSource;
         }
 
+        // Parse the emulator host and port if the SPANNER_EMULATOR_HOST environment variable is set.
+        private void ParseEmulatorHostAndPort()
+        {
+            // Emulator host and port have already been parsed.
+            if (!string.IsNullOrEmpty(_emulatorHost))
```

Comment:
nit: only need to check for null here, all instance variables are initialized to the type's default value and that is null for string, which is a reference type.